pkg/profile/artifact.go

```diff
@@ -126,13 +139,52 @@ func (p *Profile) createHelmRelease(ctx context.Context) error {
 	return p.client.Create(ctx, r)
 }
 
+func (p *Profile) makeKustomization() (*kustomizev1.Kustomization, error) {
+	kustomization := &kustomizev1.Kustomization{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      p.makeArtifactName(),
+			Namespace: p.subscription.ObjectMeta.Namespace,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind:       kustomizeKind,
+			APIVersion: kustomizeAPIVersion,
```

Comment:
think these can be done like `kustomizev1.KustomizeKind` and `kustomizev1.GroupVersion.String()` or something. merge in from main and you will see the thing chetan did

Reply:
:+1: will do before merging